mappings/net/minecraft/client/gl/DummyUniform.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS none/bwj net/minecraft/client/gl/DummyUniform
+	METHOD a setFloat1 (F)V
+	METHOD a setFloat2 (FF)V
+	METHOD a setFloat3 (FFF)V
+	METHOD a setFloat4 (FFFF)V
+	METHOD a setMatrix4 (FFFFFFFFFFFFFFFF)V
```

Comment:
`setMatrix4f` to go with the name of the type? *Could* also just be `setMatrix`.